register/register.py

```diff
@@ -7,10 +7,10 @@
 def lambda_handler(event, context):
     to_number = event['To']
     from_number = event['From']
-    logging.info(f'Received registration request for {from_number}')
+    logging.info('Received registration request for {from_number}')
```

Comment:
These weren't 'stray' -- https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python

But if it worked without it, that's great, not sure how it's doing the string formatting though.

Reply:
I screwed-up....I now see what the 'f's do. I'll put these back.